CircularSeek/ViewController.swift

```diff
@@ -8,7 +8,12 @@
 
 import UIKit
 
-class ViewController: UIViewController {
+protocol VcDelegate {
```

Comment:
@andydonzelli You have created a delegate in `ViewController`and exposing that to `CircularSeeker` which makes the `ViewController` and `CircularSeeker` tightly coupled. It supposed to be other way around, `CircularSeeker` have to create a delegate and expose it to the implementer, that way you can make ``CircularSeeker` loosely coupled.